modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,260 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// TBD what action must the user take?
+// link:https://github.com/tektoncd/pipeline/pull/4786
+
+* This update adds a `SkippingReason` value to the `SkippedTasks` field of the `PipelineRun` status fields so that users know why a given PipelineTask was skipped.
+// link:https://github.com/tektoncd/pipeline/pull/4829
+
+* This update supports an alpha feature in which you can use an `array` type for emitting results from a `Task` object. The result type is changed from `string` to `ArrayOrString`. For example, a task can specify a type to produce an array result:
++
+[source,yaml]
+----
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: write-array
+  annotations:
+    description: |
+      A simple task that writes array
+spec:
+  results:
+    - name: array-results
+      type: array
+      description: The array results
+...
+----
++
+And the task script can populate the result in an array form with:
++
+[source,terminal]
+----
+$ echo -n "[\"hello\",\"world\"]" | tee $(results.array-results.path)
+----
++
+This feature is in progress and is part of TEP-0076.
+// link:https://github.com/tektoncd/pipeline/pull/4818
+
+[id="triggers-new-features-1-8_{context}"]
+=== Triggers
+
+* This update transitions the `TriggerGroups` field in the `EventListener` specification from being an alpha feature to being a stable feature. Using this field, you can specify a set of interceptors before selecting and running a group of triggers. It is available by default, and you do not need to change the configuration to use it.
+// link:https://github.com/tektoncd/triggers/pull/1345(#1345)
+// @dlorenc
+
+* With this update, the `Trigger` resource supports end-to-end secure connections by running the `ClusterInterceptor` server using HTTPS.
+// link:https://github.com/tektoncd/pipeline/pull/1333(#1333)
+// @savitaashture
+
+[id="cli-new-features-1-8_{context}"]
+=== CLI
+
+* With this update, you can use the `tkn taskrun export` command to export a live task run from a cluster to a YAML file, which you can use to import the task run to another cluster.
+// link:https://github.com/tektoncd/cli/pull/1531[(#1531)]
+// Chmouel Boudjnah @chmouel
+
+* With this update, you can add the `-o name` flag to the `tkn pipeline start` command to print the name of the pipeline run right after it starts.
+// link:https://github.com/tektoncd/cli/pull/1540[(#1540)]
+// Chmouel Boudjnah @chmouel
+
+* This update adds a list of available plugins to the output of the `tkn --help` command.
+// link:https://github.com/tektoncd/cli/pull/1535[(#1535)]
+// Chmouel Boudjnah @chmouel
+
+* With this update, while deleting a pipeline run or task run, you can use both the `--keep` and `--keep-since` flags together.
+// link:https://github.com/tektoncd/cli/pull/1533[(#1533)]
+// Divyanshu Agrawal @divyansh42
+
+* With this update, you can use `Cancelled` as the value of the `spec.status` field instead of the deprecated `PipelineRunCancelled` value.
```

Comment:
```suggestion
* With this update, you can use `Cancelled` as the value of the `spec.status` field rather than the deprecated `PipelineRunCancelled` value.
```
Per Vale - Use `rather than` rather than `instead of`

Reply:
Lol: I don't know where that Vale rule came from.  It's not present in any of our style guides. I'll have to follow up in with the Vale team.